cves/CVE-2016-2161.yml

```diff
@@ -220,4 +230,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulenrability is the result of a coding mistake. More specifically, a missed edge case 
+    that can be used maliciously for a dos attack. The vulnerability was created in a single commit but
+    the allocation functions had been rewritten a few times previously. It does not appear that prior functions
+    covered this edge case either. 
+
+    This vulenrability falls under the Availability category in the CIA model and is classified as a High threat
+    due to it's ability to bring down the system. The fix appeals to an input validation strategy and distrusts the memory allocation of another function.
```

Comment:
Interesting that you mention distrusting another function that deals with memory. 
Could this also be related to defensive coding in depth?

Reply:
It definitely does - nice suggestion! Added it.